examples/todoapp/lib/lib_realm/services/realm_service.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:realm/realm.dart';
+
+import '../../base/data_sources/local/connectivity_data_source.dart';
+import '../../base/models/todo_model.dart';
+import '../repositories/realm_init_repository.dart';
+
+class RealmService {
+  RealmService(this.realmInitRepository);
+
+  static const String queryAllTodos = 'getAllTodos';
+  static const String _realmAppId = 'remindersservice-rlxovpu';
```

Comment:
Please move the `_realmAppId` to the env configuration

Reply:
Done